event/event.go

```diff
@@ -1,32 +1,34 @@
 package event
 
 import (
-	"calendar/user"
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/google/uuid"
+	"gorm.io/gorm"
 	"time"
```

Comment:
Best practice is to sort import - 
STD libs
empty line
3-d party libs
empty line
your project inner dependencies

goimport should sort it for you if setup correctly